src/main/java/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import service.CsvFileWriterService;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {

```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line above the class declaration to follow Java conventions.